app/javascript/components/tree-view/hierarchical-tree-view.jsx

```diff
@@ -0,0 +1,80 @@
+/* eslint camelcase: ["warn", {allow: ["bs_tree", "tree_name", "click_url", "check_url", "allow_reselect"]}] */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Tree, { ActionTypes } from '@manageiq/react-ui-components/dist/wooden-tree';
+
+const convertData = data => Tree.convertHierarchicalTree(Tree.initHierarchicalTree(JSON.parse(data)));
+
+function HierarchicalTreeView(props) {
```

Comment:
```const { check_url, oncheck, bs_tree } = props;```

up here instead of line 12, that should make the linter happier.

Reply:
thanks